### DISCUSSION TITLE
extraction Issue

### DISCUSSION THREAD

**Discussion by hojjatjafary** [gildor2/UEViewer#244] [General]
I get this error message, what does it mean?

appDecompress: unknown compression flags 10, detected 2, retrying ...
ERROR: lzo_decompress(12350,65536) returned -6
appDecompress: CompSize=12350 UncompSize=65536 Flags=0x10 Bytes=8C86

**Comment by gildor2** [comment]
You should post FULL error message. Plus, usually people at least says which game they're trying to decompress.

But answering to your question - the error means that compressed data has some custom format, umodel tried to auto-detect it, and failed.

**Reply by hojjatjafary** [reply]
I'm using the tool for Ashen game that developed using Unreal 4.19, and here is full error message:
```
appDecompress: CompSize=623 UncompSize=926 Flags=0x10 Bytes=8C86 <- SerializeCompressed <- FPakFile::Serialize: file=/Engine/Maps/Templates/Thumbnails/Default.uasset <- FArchive::ByteOrderSerialize <- UnPackage::CreateLoader: /Engine/Maps/Templates/Thumbnails/Default.uasset <- UnPackage::UnPackage: /Engine/Maps/Templates/Thumbnails/Default.uasset, ver=100000/0, game=0 <- UnPackage::LoadPackage(info): /Engine/Maps/Templates/Thumbnails/Default.uasset <- UIMenuItem::HandleCommand <- UIMenu::Popup <- UIBaseDialog::WndProc <- UIBaseDialog::CustomMessageLoop <- UIBaseDialog::ShowDialog: modal=1, title="Choose a package to open" <- CUmodelApp::ShowPackageUI <- Main: umodel_build=1573
```

**Comment by gildor2** [comment]
Read this
https://www.gildor.org/smf/index.php/topic,6451.0.html

**Reply by hojjatjafary** [reply]
Thank you so much, resolved.

**Comment by gildor2** [comment]
By the way, if you'd share any (small) pak file with me, I'd try to fix UModel - it definitely works in a way I didn't suppose to do.
